app/server/tests/test_social_auth.py

```diff
@@ -93,3 +95,75 @@ def test_fetch_permissions_not_admin(self):
         )
 
         self.assertFalse(user.is_superuser)
+
+
+@override_settings(SOCIAL_AUTH_OKTA_OAUTH2_KEY='0000000000aaaaaaaaaa')
+@override_settings(SOCIAL_AUTH_OKTA_OAUTH2_SECRET='bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb=')
+@override_settings(SOCIAL_AUTH_OKTA_OAUTH2_API_URL='https://dev-000000.okta.com/oauth2')
+@override_settings(OKTA_OAUTH2_ADMIN_GROUP_NAME='admin-group')
+class TestOktaOAuth2SocialAuth(VCRTestCase):
+    strategy = None
+    backend = OktaOAuth2(strategy=strategy)
+    access_token = 'censored'
+
+    def test_fetch_permissions_is_admin(self):
+        user = User()
+
+        social_auth.fetch_okta_oauth2_permissions(
+            strategy=self.strategy,
+            details={},
+            user=user,
+            backend=self.backend,
+            response={'access_token': self.access_token},
+        )
+
+        self.assertTrue(user.is_superuser)
+
+    def test_fetch_permissions_not_admin(self):
+        user = User()
+
+        social_auth.fetch_okta_oauth2_permissions(
+            strategy=self.strategy,
+            details={},
+            user=user,
+            backend=self.backend,
+            response={'access_token': self.access_token},
+        )
+
+        self.assertFalse(user.is_superuser)
+
+
+@override_settings(SOCIAL_AUTH_OKTA_OPENIDCONNECT_KEY='0000000000aaaaaaaaaa')
+@override_settings(SOCIAL_AUTH_OKTA_OPENIDCONNECT_SECRET='bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb=')
```

Comment:
You should be able to suppress the codacy false positive by adding `# nosec` to this line (see [docs](https://github.com/PyCQA/bandit#exclusions)).

Reply:
Didn't know that, nice. Thanks!